docs/docs/quickstart.mdx

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 3
+title: Quickstart
+---
+
+# Quickstart
+
+Follow these steps for a quick start:
+
+1. Install Chainloop. This quick snippet will get you through the process:
+    ```bash
+    curl -sfL https://docs.chainloop.dev/install.sh | bash -s
+    ```
+    Refer to [these instructions](/getting-started/installation) for more installation options.
+
+2. Authenticate to the Control Plane via OIDC by running:
+    ```bash
+    chainloop auth login
+    ```
+    This will create a personal token that you will use in operations not related to attestations.
+
+3. Create a new Chainloop workflow.
+    ```bash
+    chainloop wf create --name mywf --project myproject
+    ```
+    Chainloop workflows represent any CI or process you might want to attest.
+    Check [this doc](/getting-started/workflow-definition)
+    for a complete explanation of Workflows and Contracts.
+    You might also want to check our [contract reference](/reference/operator/contract).
+
+4. Create a restricted token for further operations:
```

Comment:
I personally wouldn't talk about `restricted token`, or `restricted permissions`, I'd just call this `Create API token to perform the attestation process`

The reason being that I think it's more important to know what is it for, that what it is, in fact, if we follow your approach, some people might opt for not to create since the user one "is supposed to work since it has a super set of permissions", and that's not the case